### ISSUE TITLE
Day 2 - Environment Setup Step 6 - Failing

### ISSUE THREAD

**Issue by ssainiuf** [Snowflake-Labs/llm-bootcamp-build-2023#30]
````sql
select 
  v.value:containerName::varchar container_name
  ,v.value:status::varchar status  
  ,v.value:message::varchar message
from (select parse_json(system$get_service_status('FINETUNE_LLM_SERVICE'))) t, 
lateral flatten(input => t.$1) v;
````

While running the above in SQL worksheet, I get the response as below - 
![error](https://github.com/Snowflake-Labs/llm-bootcamp-build-2023/assets/90658065/62f084cf-2ee9-4d01-9b4b-db7a8983b606)

I did not run Step 4 due to #6

**Comment by zarez** [comment]
Same here.

**Comment by ssainiuf** [comment]
@zarez I thought it may have been due to a failed docker push as seen in #35. But if you are facing the same issue even after a successful push, then I have no clue what's happening.

**Comment by zarez** [comment]
Yes, after a successful push. The problem looks like it is llm-bootcamp.yaml. Check the image path. I think we need rebuild and push again 
`spec:
  container:
  - name: llm-bootcamp
    image: sfsenorthamerica-build-spcs.registry.snowflakecomputing.com/db_user####/schema_llm/image_repo/llm-bootcamp
    volumeMounts:
    - name: llm-workspace
      mountPath: /notebooks/llm-workspace
    env:
      SNOWFLAKE_MOUNTED_STAGE_PATH: /notebooks/llm-workspace
    resources:
        requests:
          nvidia.com/gpu: 1
        limits:
          nvidia.com/gpu: 1
  endpoint:
  - name: llm-bootcamp
    port: 8888
    public: true
  volume:
  - name: llm-workspace
    source: "@llm_workspace"
    uid: 0
    gid: 0`

**Comment by zarez** [comment]
FIXED that one and stumble on next one: 

- Ensure you update the **account name** and **db user** in the **llm-bootcamp.yaml**
- You need snowsql to connect to DB. 
`
snowsql -a sfsenorthamerica-build_spcs10 -u db_user0016
`
- set db, schema and warehouse
- execute
`
PUT file:///Users/gjorgji/code/gh/snowflake/llm-bootcamp-build-2023/day2/llm-bootcamp.yaml @YAML_STAGE AUTO_COMPRESS=FALSE OVERWRITE = TRUE;
`

- drop and recreate the service. 

Now my service says: PENDING	Waiting to start

**Comment by zarez** [comment]
Scrap that. Its working, just needed more time to start
![image](https://github.com/Snowflake-Labs/llm-bootcamp-build-2023/assets/772740/3533c50b-6d39-4c1b-88fa-6e3dc5ec0aac)

**Comment by qusaikagalwala** [comment]
3) Replace ***your-account-name*** with your account name and ***your-db-name*** with the name of your database ***in lowercase*** and then run the following command
    
        `docker push sfsenorthamerica-build_spcs9.registry.snowflakecomputing.com/db_user0021/schema_llm/image_repo/llm-bootcamp:latest` --- Could not run this command on Docker?

**Comment by zarez** [comment]
Change underscore to hyphen in **sfsenorthamerica-build_spcs9**
